src/lib/components/EmailDetailView.svelte

```diff
@@ -216,8 +216,48 @@
 									</div>
 								</div>
 							</div>
+							<div>
+								<div role="listbox" aria-label="file attachments" class="attachment-list">
+									{#each email.attachments as attachment}
+										<a
+											href={attachment.url}
+											download
+											title="Download"
+											class="attachment-item"
+											role="option"
+											aria-label={attachment.name}
+										>
+											<!-- Icon -->
+											<img class="attachment-icon" src={attachment.url} />
```

Comment:
The icon should not be the URL. It should be based on the file type from Iconify